src/script/components/UserList/components/UserListItem/UserListItem.tsx

```diff
@@ -26,149 +26,115 @@ import {Checkbox, CheckboxLabel} from '@wireapp/react-ui-kit';
 
 import {AvailabilityState} from 'Components/AvailabilityState';
 import {Avatar, AVATAR_SIZE} from 'Components/Avatar';
-import {ParticipantMicOnIcon} from 'Components/calling/ParticipantMicOnIcon';
 import {Icon} from 'Components/Icon';
 import {UserlistMode} from 'Components/UserList';
 import {InViewport} from 'Components/utils/InViewport';
 import {useKoSubscribableChildren} from 'Util/ComponentUtil';
-import {KEY} from 'Util/KeyboardUtil';
 import {t} from 'Util/LocalizerUtil';
 import {capitalizeFirstChar} from 'Util/StringUtil';
-import {noop, setContextMenuPosition} from 'Util/util';
 
-import {Participant} from '../../calling/Participant';
-import {User} from '../../entity/User';
-import {ServiceEntity} from '../../integration/ServiceEntity';
+import {User} from '../../../../entity/User';
 
-export interface ParticipantItemProps<UserType> extends Omit<React.HTMLProps<HTMLDivElement>, 'onClick' | 'onKeyDown'> {
+export interface UserListItemProps {
   badge?: boolean;
```

Comment:
Let's rename this prop to sth like `hasBadge` since it's `boolean` type.

Reply:
Or actually seems like it was storing the same value as `highlighted` prop. Let's name it `isHighlighted`.